kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -77,6 +80,75 @@ var StatusModel = Backbone.Model.extend({
         return (new Date(new Date() - this.get("client_server_time_diff"))).toISOString().slice(0, -1);
     },
 
+    login: function(username, password, facility, callback) {
+        // Add a callback to allow functions calling this method to do things depending on the result of
```

Comment:
I think it will be better if we expand this into a full description of this function, specifically what types are expected for each parameter, and how is it expected to be called.